.travis.yml

```diff
@@ -2,6 +2,7 @@ language: cpp
 os: linux
 dist: xenial
 env: WORKSPACE=$TRAVIS_BUILD_DIR
+cache: ccache
```

Comment:
ccache only makes sense when you can share it between builds. But this is a docker build which is initialized new on each trigger so you will basically always have a cold cache.